### DISCUSSION TITLE
How can I fly crazyflie with ROS without a  joystick?

### DISCUSSION THREAD

**Discussion by chenxlin222** [USC-ACTLab/crazyswarm#546] [Q&A]
I have read the tutorial in http://act.usc.edu/publications/Hoenig_Springer_ROS2017.pdf , but I realize that it need a joystick to teleoperate Crazyflie with ROS. How can I fly crazyflie with ROS without a  joystick?  Because I don't have a  Microsoft XBox360 or other controller. Thanks very much!!!

**Comment by whoenig** [comment]
The Crazyswarm does not use (or need) a Joystick. You can find an example without a joystick at https://crazyswarm.readthedocs.io/en/latest/tutorials/tutorials.html.
